components/Storyblok/SbMegaMenu/SbMegaMenuCard.tsx

```diff
@@ -40,21 +39,19 @@ export const SbMegaMenuCard = ({ blok }: SbMegaMenuCardProps) => {
       )}
       <div className={styles.cardContent}>
         {headline && (
-          <Heading className={styles.cardHeading}>
-            <SbLink link={link} className={styles.headingLink}>
-              {headline}
-            </SbLink>
+          <Heading color="white" className={styles.cardHeading}>
+            {headline}
           </Heading>
         )}
         {ctaText && (
-          <Paragraph
-            color="white"
-            weight="semibold"
-            icon={link?.linktype === 'url' ? 'external' : 'chevron-right'}
-            iconProps={{ className: styles.cardCtaTextIcon }}
+          <CtaLink
+            sbLink={link}
+            icon={link?.linktype === 'url' ? 'su-link--external' : 'su-link--action'}
+            textColor="white"
+            className={styles.cta}
           >
             {ctaText}
-          </Paragraph>
+          </CtaLink>
```

Comment:
For the mega menu card, instead of putting the stretched link in the heading, put the link around the CTA text